api/v1beta1/cloudstackmachinestatechecker_types.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// CloudStackMachineStateCheckerSpec defines the desired state of CloudStackMachineStateChecker
+type CloudStackMachineStateCheckerSpec struct {
+	// Give better description.
```

Comment:
@Wonkun! Haha

Reply:
Also the comment above it no longer makes sense really. Please update that as well.